docs/io/unified.rst

```diff
@@ -318,6 +316,15 @@ the keywords ``DATE``, ``DATE-*`` (ISO-8601 datetime strings) and the ``MJD-*``
 For more details regarding the FITS time paper and the implementation,
```

Comment:
This is really a comment for line 310 (for `QTable`). I think it should have a bit of extra text, as follows. Given that, I also think it makes sense to have ``astropy_native=True`` as the default for `QTable`...
```
Note that even with ``astropy_native=True``, numerical columns with units will
be stored as `~astropy.table.Column` instances, i.e., they will not be
converted to `~astropy.units.Quantity` objects. For that, one should use
`~astropy.table.QTable`; for that class, ``astropy_native`` is set to `True`
by default.
```